coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -4607,6 +4607,10 @@ def _scatter(context, inputs, mode, name):
     axis = inputs[1].val
     indices = inputs[2]
     updates = inputs[3]
+    if not types.is_tensor(updates.sym_type):
+        # torch.scatter allows for src (updates) argument to be either tensor or scalar
```

Comment:
Thanks for explaining this information. However I don't think we need the comment. I think the code is clear enough. Please delete the comment.